src/components/App.tsx

```diff
@@ -53,6 +59,31 @@ class App extends React.Component<Record<string, unknown>, AppState> {
       </div>
     );
   }
+
+  private async fetchAave(address, ethPrice) {
+    console.log("fetching aave for account " + this.mainAccount);
+    let lowercaseAddress = address.toLowerCase();
+    const v2Reserves = await AaveClient.query({
+      query: V2_RESERVES,
+      fetchPolicy: "cache-first",
+    });
+    const v2UserReserves = await AaveClient.query({
+      query: V2_USER_RESERVES,
+      variables: {
+        id: lowercaseAddress,
+      },
+      fetchPolicy: "cache-first",
+    });
+    let usdPriceEth = (1 / ethPrice) * 1000000000000000000;
+    let v2UserSummary = v2.formatUserSummaryData(
+      v2Reserves.data.reserves,
+      v2UserReserves.data.userReserves,
+      lowercaseAddress,
+      usdPriceEth,
+      Math.floor(Date.now() / 1000)
+    );
+    console.log("v2 user summary: ", v2UserSummary);
+  }
```

Comment:
optional/nit: we could probably create a file just for these aave functions like an `aave-util` or something.

Reply:
similarly to how you created `AaveClient` so we can keep `App.tsx` to just focus on state of component/visual aspects of component. When we're done with technical stuff, we can make `App.tsx` a smart component that just renders a bunch of dumb components (https://medium.com/@thejasonfile/dumb-components-and-smart-components-e7b33a698d43)

Reply:
yep, I'm down to do that as a follow-up! since this is getting big